mava/configs/network/transformer.yaml

```diff
@@ -0,0 +1,6 @@
+# --- Network params ---
+n_block: 1 # Transformer blocks
+n_embd: 64 # Transformer embedding dimension
```

Comment:
for sake of unification maybe we can rename this to embed_dim similar to Sable

Reply:
Done.